modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,116 @@
+= Mule Upgrade Tool
+
+*Latest version: 1.0.0*
+
+include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool.adoc[tag=intro]
+
+The Mule upgrade tool runs in any environment that is capable of running a standalone Mule instance. This tool can update only standalone Mule instances that are running any Mule 4 version. You can upgrade local Mule instances to
+//the latest available version or to
+a particular Mule distribution that you have previously downloaded and unpacked. You can also use the Mule upgrade tool to update Mule instances located in different folders.
+
+As part of the upgrade process, the Mule upgrade tool also updates the Runtime Manager agent version if it is installed in your current Mule instance.
+
+== Before You Begin
+
+Before starting your upgrade, ensure that you have:
+
+* A currently operational Mule 4 instance, in _stopped_ status to prepare for the upgrade
+* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory
++
+If the Mule upgrade tool is not bundled with your current Mule distribution, download the tool from the https://help.mulesoft.com/s/[Support Portal^] and copy it into the `$MULE_HOME/tools` directory of your current Mule instance.
+* The new Mule runtime engine distribution you want to upgrade to
++
+You can download Mule distributions from the https://help.mulesoft.com/s/[Support Portal^].
+* At least 2 GB of available disk space on the file system and access privileges to install the new Mule distribution.
+//Confirm if the tool is downloaded from the Support portal or help center, and update the link.
+
+//This feature is not yet available
+////
+== Upgrade Mule to the Latest Version
+
+To upgrade your Mule instance to the latest version:
+
+. Navigate to the Mule upgrade tool directory.
+. Run the `upgrade` command:
++
+```
+$MULE_HOME/tools/upgrade-tool upgrade
+```
+////
+
+== Upgrade Mule to a Specific Distribution
+
+To upgrade your Mule instance to a specific Mule distribution:
```

Comment:
Same here, we might document this as the need to specify the path to where the new distribution is located, but not aimed to upgrade to a "specific" distribution. Although it is possible, we should encourage latest only.
cc @estebanwasinger

Reply:
Yes, despite that we are not doing any kind of version control, I would rephrase it to make it clear that you should always update to the "latest version"

Reply:
Addressed in [d6fd62a](https://github.com/mulesoft/docs-mule-runtime/pull/2159/commits/d6fd62ae387578659d4972550cd9f16a41264be1)